File: NuGet.Package/content/Properties/MocaAssemblyInfo.cs

[assembly: WebActivatorEx.PreApplicationStartMethod(typeof(Moca.Di.MocaContainerFactory), "Init", Order = 1)]
[assembly: WebActivatorEx.PreApplicationStartMethod(typeof(Moca.Web.Mvc.MocaControllerFactory), "Init", Order = 2)]
[assembly: WebActivatorEx.PreApplicationStartMethod(typeof(Moca.Configuration.SectionProtector), "Protect", Order = 3)]
[assembly: WebActivatorEx.ApplicationShutdownMethod(typeof(Moca.Di.MocaContainerFactory), "Destroy")]

File: NuGet.Package/content/My Project/MocaAssemblyInfo.vb

<Assembly: WebActivatorEx.PreApplicationStartMethod(GetType(Moca.Di.MocaContainerFactory), "Init", Order:=1)>
<Assembly: WebActivatorEx.PreApplicationStartMethod(GetType(Moca.Web.Mvc.MocaControllerFactory), "Init", Order:=2)>
<Assembly: WebActivatorEx.PreApplicationStartMethod(GetType(Moca.Configuration.SectionProtector), "Protect", Order:=3)>
<Assembly: WebActivatorEx.ApplicationShutdownMethod(GetType(Moca.Di.MocaContainerFactory), "Destroy")>
